src/Stratis.Bitcoin/Base/BaseFeature.cs

```diff
@@ -299,6 +306,8 @@ public override void Dispose()
             }
 
             this.consensusRules.Dispose();
+
+            (this.blockStore as IDisposable)?.Dispose();
```

Comment:
As before, I would probably prefer Idisposable on the interface itself

Reply:
Ok yeah can do that.